modules/rosa-getting-started-verify-elb-role.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started/rosa-getting-started.adoc
+
+[id="rosa-getting-started-verify-elb-role_{context}"]
+= Creating the ELB service role
+
+Check if the `AWSServiceRoleForElasticLoadBalancing` AWS Elastic Load Balancing (ELB) service role exists and if not, create it.
+
+[NOTE]
+====
+An `Error creating network Load Balancer: AccessDenied:` error is produced if you attempt to create a {product-title} (ROSA) cluster without the AWS ELB service role in place.
```

Comment:
Consider:
"`Error creating network Load Balancer: AssessDenied:` appears if you ..."

Reply:
Great suggestion, thank you! Applied.